pygccxml/parser/source_reader.py

```diff
@@ -146,9 +145,13 @@ def __create_command_line_castxml(self, source_file, xmlfile):
             else:
                 std_flag = ' ' + self.__cxx_std.stdcxx + ' '
 
+            ccflags = self.__config.ccflags
             if std_flag:
+                ccflags += std_flag
+
+            if ccflags:
                 cmd.append(
-                    '"(" ' + self.__config.compiler_path + std_flag + '")"')
+                    '"(" ' + self.__config.compiler_path + ccflags + ' ")"')
```

Comment:
`compiler_path` and `ccflags` should be separated by a space

```
 '/usr/bin/c++-std=c++14' '-DF0_NODE_TEMP' '-DF0_UPDATE_ANALYTIC_OLD' '-DITER_GRID' '-DSOLVERLU' '-DUSE_BICUB_MOD' '-DUSE_ONE_D_I_CUB_MOD' '-DV_PERP' '-DEXPLICIT_EM_NONLIN_TERMS' '-DMESH_BATCH_SIZE=14' '-DFFTW3' '-fopenmp=libomp' '-fopenmp' '-E' '-dM' '-v' '/usr/local/lib/python3.6/dist-packages/castxml/data/share/castxml/empty.cpp'
Executable "/usr/bin/c++-std=c++14" doesn't exist!
```

Reply:
Good catch, see 053d3f35efc422c45229004c6edc7cd1df3a2f16